pkg/reconciler/events/event.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package events
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"knative.dev/pkg/apis"
+)
+
+const (
+	// Emit events if user provides enable-eventlistener-events: "true"
+	EnableEventListenerEvents = "enable-eventlistener-events"
+	// TriggerProcessingStarted is sent for Sink triggers received which
+	// are being processed
+	TriggerProcessingStartedV1 = "triggers.tekton.eventlistener.triggerprocessing.started.v1"
```

Comment:
I'm glad that this is consistent with cloudevents but I had a comment that those names should be similar to the ones for Pipelines - see https://github.com/tektoncd/triggers/pull/1259#discussion_r759553630

Reply:
Updated :+1: